content/faq/how-do-I-configure-cloudflare-integration.md

```diff
@@ -0,0 +1,79 @@
+---
+title: How do I configure Cloudflare integration?
+menu:
+  main:
+    parent: 'faq'
+    weight: 110
+weight: 110
+linkTitle: Cloudflare Integration
+---
+
+You may want to integrate your endpoints with the [Cloudflare](https://www.cloudflare.com/) service. At a minimum this allows you to take advantage of Cloudflare DNS functionality and view deployed tides using a domain name (rather than just an IP address). If you choose to proxy the traffic as well, you can use advanced features in Cloudflare, including adding SSL to the endpoint.
+
+An example of the Cloudflare endpoint integration is as follows:
+
+```yaml
+tasks:
+    # ...
+    deployment:
+        deploy:
+            services:
+                web:
+                    specification:
+                        # ...
+                        endpoints:
+                            - name: web
+                              cloud_flare_zone:
+                                  zone_identifier: ${CLOUD_FLARE_ZONE}
+                                  authentication:
+                                      email: ${CLOUD_FLARE_EMAIL}
+                                      api_key: ${CLOUD_FLARE_API_KEY}
+                                  proxied: true
+                                  record_suffix: -test.continuouspipe.net
```

Comment:
Can you replace `continuouspipe.net` with `your-domain-name.com` or something like that ?